### DISCUSSION TITLE
Too few ovarlapped peaks between `MACS2` and `Tiles`500

### DISCUSSION THREAD

**Discussion by jiangpuxuan** [GreenleafLab/ArchR#1608] [Questions / Documentation]
I ran my data and check the converse  of  percent of TileMatrix-called peaks that are overlapped by MACS2-called peaks, just like the tutorial 10.3.1. 
There is 101547 peaks found by `Tiles`, 74562 by `MACS2`, but only 40098 overlaps. 
I `resize()` the Tile to 1000, but `Tiles` still found the same 101547 peaks.
How could I interpret this situation?

Here is my archR project: 

```
class: ArchRProject 
outputDirectory: /Array-Tuqiang/pxjiang/analysis/atac/CRC_P07A/proj_crc07_hg19 
samples(1): Human_0721
sampleColData names(1): ArrowFiles
cellColData names(34): Sample TSSEnrichment ... ReadsInPeaks FRIP
numberOfCells(1): 1364
medianTSS(1): 6.142
medianFrags(1): 3717
```
Thank you for your help!

**Comment by rcorces** [comment]
I would not use the Tiles method for peak calling. I'm making this more clear in the next documentation release. It was really only provided for people who had issues installing MACS2 and it has not been extensively tested.

**Reply by jiangpuxuan** [reply]
> I would not use the Tiles method for peak calling. I'm making this more clear in the next documentation release. It was really only provided for people who had issues installing MACS2 and it has not been extensively tested.

Thank you for your help!
